### DISCUSSION TITLE
In the future, will something like 'remote variable' be added?

### DISCUSSION THREAD

**Discussion by JiaPai12138** [keygen-sh/community#45] [Q&A] [answered]
For example, if i want to store some encryption keys or contents remotely instead of locally to increase the difficulty of cracking my software...

**Comment by ezekg** [comment, marked as answer]
You could use the distribution API to store files and other artifacts. Or you could add this information to a license’s metadata. But encrypting with a “secret” like this sounds like security through obscurity, unless the keys are per-license. Somebody could easily share the key and then you’re back at square 1.
